### DISCUSSION TITLE
RTDs not reading correctly

### DISCUSSION THREAD

**Discussion by raymop** [tagyoureit/nodejs-poolController#485] [REM] [answered]
I installed 10k ohm RTDs per the [guide](https://github.com/rstrouse/relayEquipmentManager/wiki/Temperature-Sensors#step-5-feed-your-temperature-probe).
Used a [16-bit ADC](https://www.amazon.com/dp/B07PNMXR8B?psc=1&ref=ppx_yo2ov_dt_b_product_details) per the guide. 
Used [RTDs](https://www.amazon.com/dp/B07NWK6GJ4?psc=1&ref=ppx_yo2ov_dt_b_product_details) per the guide.
Configured all the steps per the guide.
Also installed a 10k OHM resistor between 3.3V and AIN0, with RTD between GND and AIN0 on the ADC.
But am not getting correct readings.
Temp outside is about 80 F, while RTD reads 59 F.

Would love to hear thoughts!!!

Here are screenshots of my configuration for REM.
![image](https://user-images.githubusercontent.com/4217310/173708004-30f27a9d-380d-486d-b5fb-00aef1cb8f56.png)
![image](https://user-images.githubusercontent.com/4217310/173708030-b1aedc76-8be1-4c7c-a364-1a93fb97cd45.png)
![image](https://user-images.githubusercontent.com/4217310/173708048-ba2e0044-b79a-4424-a518-e42d63083910.png)
![image](https://user-images.githubusercontent.com/4217310/173708148-ceed1ba7-1965-44cc-9bab-d186ba763975.png)
![image](https://user-images.githubusercontent.com/4217310/173708186-fdd3b424-12d6-453e-aec8-ac9e9eadc215.png)
![image](https://user-images.githubusercontent.com/4217310/173708225-0bea4ce6-87f7-4b59-aadf-dccfc41cef79.png)
![image](https://user-images.githubusercontent.com/4217310/173708285-e8e93da0-bbe0-4a10-aedf-070bfa7d2adf.png)
![image](https://user-images.githubusercontent.com/4217310/173708404-19eecda5-4121-4066-915d-d240a0c72bf1.png)

**Comment by rstrouse** [comment, marked as answer]
There is a jumper on that board that will hijack AIN0.  If you have the jumper to enable NTC on AIN0, remove it.

**Reply by raymop** [reply]
That helped. I also had to select a different address on the i2c bus by moving the jumper.

**Reply by raymop** [reply]
Thank you!!!!
